### PR TITLE
[12.0][MIG] stock_inventory_virtual_location

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

File: stock_inventory_virtual_location/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': "Stock Inventory Virtual Location",
+    'summary': "Allows to change the virtual location"
+               " in inventory adjustments.",
+    'author': 'Eficent, Odoo Community Association (OCA)',
+    'website': "https://github.com/OCA/stock-logistics-warehouse",
+    'category': 'Warehouse Management',
```
Comment (rousseldenis) [inline stock_inventory_virtual_location/__manifest__.py]: Could you set this to 'Warehouse' as it won't appear in OCA shop in the good category?

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review.

Could you make another PR for the second module ?

File: stock_inventory_virtual_location/__manifest__.py (hunk 1 above)
Comment (MiquelRForgeFlow) [reply stock_inventory_virtual_location/__manifest__.py]: done for all modules

**Comment by mreficent** [comment]
> Could you make another PR for the second module ?

No, too lazy/busy to do that now.

**Comment by MiquelRForgeFlow** [comment]
> Could you make another PR for the second module ?

@rousseldenis why? I think it's not necessary in this specific case.

**Review by HviorForgeFlow — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]

**Comment by rousseldenis** [comment]
@MiquelRForgeFlow Could you rebase ?

**Review by marcelsavegnago — APPROVED** [review]

**Comment by MiquelRForgeFlow** [comment]
@rousseldenis rebased

**Review by HaraldPanten — APPROVED** [review]

**Pull request merged**